linux-create-vm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Install git
+  echo "git should be installed"
+if ( which git )
+then 
+  echo "git already installed"
+else 
+  echo "installing git"
+  sudo apt install git
+fi
+
+#install nano
+  echo "nano should be installed"
+if ( which nano )
+then
+  echo "nano already installed"
+else
+  echo "installing nano"
+  sudo apt install nano 
```

Comment:
does this generate a prompt?

Reply:
it does not

Reply:
it does not

Reply:
it will actually, you need the `install -y nano`